Automattic/wp-calypso
# Google My Business: Refine all pages

This pull request seeks to address all the points raised during the design review with @cburton4 on the Google My Business flow. More specifically, it:

* Highlights the `Stats` menu item in the sidebar when the Google My Business `Stats` page is displayed
* Sets interval drop-downs width to 100% on small viewports
* Removes line separators that were located above these interval drop-downs
* Removes all three tip sections from the page
* Reduces font size of pie chart's title and legend (and use sizes that are consistent with line charts)
* Reduces the pie chart size, and makes it proportional to the width of the page
* Displays the `Stats` page on two columns starting from 1040px instead of 1280px
* Centers empty data notices vertically
* Refines margins around legends and charts
* Adds a legend to the `LineChart` example in the Devdocs

##### Stats page

Before | After
------ | -----
<img width="365" alt="screenshot" src="https://user-images.githubusercontent.com/594356/40495366-5525c22a-5f77-11e8-94a4-e15e3409d687.png">| <img width="369" alt="screenshot" src="https://user-images.githubusercontent.com/594356/40495425-76ad74c4-5f77-11e8-9460-8420c3255e50.png">

##### Line Chart Example page

Before | After
------ | -----
<img width="374" alt="screenshot" src="https://user-images.githubusercontent.com/594356/40495486-98bac59e-5f77-11e8-98b3-9221a3554fd5.png"> | <img width="374" alt="screenshot" src="https://user-images.githubusercontent.com/594356/40495505-a89c6512-5f77-11e8-83af-ed8a3fa1c18a.png">

##### Pie Chart Example page

Before | After
------ | -----
<img width="369" alt="screenshot" src="https://user-images.githubusercontent.com/594356/40495548-be8a0050-5f77-11e8-9dd0-372580f74cc0.png"> | <img width="374" alt="screenshot" src="https://user-images.githubusercontent.com/594356/40495572-d0232940-5f77-11e8-9d26-40d5f9f66aca.png">
 
#### Testing instructions

As a reminder, the _empty data notice_ is the one with the `No activity this period` label:

1. Run `git checkout update/google-my-business-stats` and start your server, or open a [live branch](https://calypso.live/?branch=update/google-my-business-stats)
2. Open the [`Stats` page](http://calypso.localhost:3000/me/stats) for a site already connected to Google My Business
3. Assert that the empty data notice displays correctly in the middle of the chart
4. Clicks the `Insights` tab in the header
5. Assert that the empty data notice in the `Most popular day and hour` section displays correctly
6. Clicks the `Google My Business` tab in the header
7. Assert that the page looks like the screenshot above
8. Assert that it still displays fine if you change the width of your browser
9. Assert that the `Stats` menu item is highlighted in the sidebar
10. Open the [`LineChart` example page](http://calypso.localhost:3000/devdocs/design/line-chart)
11. Assert that it looks like the screenshot above
10. Open the [`PieChart` example page](http://calypso.localhost:3000/devdocs/design/pie-chart)
11. Assert that it looks like the screenshot above

#### Additional notes
 
During the design review, it was mentioned that we should move the pie chart to the left on small viewports (as it was initially designed in the mockup). I gave it a try but found it looked better centered even in that case. I'm happy to make that change if we feel differently.

#### Reviews
